### ISSUE TITLE
Breadcrumbs show incorrectly when doing drilldown for Chrome v. 126.0

### ISSUE THREAD

**Issue by linhphamrnd4u** [highcharts/highcharts#21344]
#### Expected behavior 
User can click on the upper level to go back after doing drilldown

#### Actual behavior
The label of upper level shows incorrectly

#### Live demo with steps to reproduce
1. Open this demo: https://www.highcharts.com/demo/highcharts/column-drilldown
2. Click drilldown on any column 
3. Click back to the upper level
4. Click on drilldown again. 

The breadcrumbs don't show fully.
It happens on newest version of Chrome 126.0. It works fine with Chrome v. 125.0 or Safari

https://github.com/highcharts/highcharts/assets/118960970/bd48dbe6-cb00-4f18-876d-0a9fa9a61096

#### Affected browser(s)
Chrome Version 126.0.6478.62 (Official Build) (arm64)

**Comment by abidibo** [comment]
Confirmed and happening also with
![image](https://github.com/highcharts/highcharts/assets/444093/d565acfa-c949-48a7-a4ea-b60414948292)

**Comment by karolkolodziej** [comment]
Hi @linhphamrnd4u and @abidibo! 

It looks like the issue has already been resolved on the Chrome end see: https://issues.chromium.org/issues/346830796
tl;dr When removing child elements from the parent the parent size is not updated.

You can workaround it by removing the `breadcrumbs` group when you do the drill down- [demo](https://jsfiddle.net/BlackLabel/1acsvnLm/).

I'll leave it open so we can check this once again after the Chrome release.

**Comment by tashapiro** [comment]
Thank you @karolkolodziej ! 

Seeing this bug on Arc browser too. I'll give the workaround a try.
